src/sbt-test/updateimpact/basic_13_8/expected.txt

```diff
@@ -110,31 +110,17 @@ com.softwaremill.example:sbt-example_2.11:0.1-SNAPSHOT in runtime
       org.apache.httpcomponents:httpclient:4.0.1
       org.scala-lang:scala-library:2.11.6
    commons-codec:commons-codec:1.3
-      junit:junit:3.8.1
```

Comment:
Turns out it is expected behavior. We do not want to include test dependecies from dependecies   of our module.